infobip_channels/web_rtc/models/body/core.py

```diff
@@ -0,0 +1,25 @@
+from typing import Optional
+
+from pydantic import constr
+
+from infobip_channels.core.models import CamelCaseModel, MessageBodyBase
+
+
+class AndroidServerKey(CamelCaseModel):
+    fcm_server_key: constr(min_length=1)
+
+
+class IosCertificates(CamelCaseModel):
+    apns_certificate_file_name: constr(min_length=1)
+    apns_certificate_file_content: constr(min_length=1)
+    apns_certificate_password: Optional[str] = None
+
+
+class ApplicationBody(MessageBodyBase):
+    name: constr(min_length=1)
+    description: Optional[constr(max_length=160)]
```

Comment:
Can you add `= None` to the end of all of the `Optional` fields just so we're consistent with the rest of the code.

Reply:
💯